sycl/test/usm/allocatorll.cpp

```diff
@@ -35,23 +33,21 @@ int main() {
   Node *d_cur = nullptr;
   Node h_cur;
 
-  for (int i = 0; i < numNodes; i++) {
-    if (i == 0) {
-      d_head = alloc.allocate(1);
-      if (d_head == nullptr) {
-        failed = true;
-        break;
-      }
-      d_cur = d_head;
-    }
+  try {
+    d_head = alloc.allocate(1);
+  } catch (memory_allocation_error e) {
+    return -1;
+  };
```

Comment:
I think you can remove exception handling from this test.
If `allocate` throws an exception, the test fill fail.

Reply:
Fixed.